db/migrate/20171021063440_add_user_status_for_disabling_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserStatusForDisablingUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :status, :integer, default: 0, null: false
```

Comment:
@gamalielhere This migration and all of the auto-generated documentation and schema are not used anymore since we removed the "status" of deleted.